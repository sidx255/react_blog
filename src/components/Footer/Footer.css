.footer {
    display: flex;
    background-color: #121212;
    padding-top: 2rem;
    padding-bottom: 2rem;
    color: white;
    bottom: 0;
    /* width: 100%; */
    padding-left: 30px;
}
  