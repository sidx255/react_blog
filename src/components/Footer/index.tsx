import React from 'react';
import './Footer.css';

const Footer = () => {
  return (
    <div className='body-padding footer'>
      &copy; Siddharth Sharma 2023
    </div>
  );
};

export default Footer;