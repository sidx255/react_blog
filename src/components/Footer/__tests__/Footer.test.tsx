import { render } from '@testing-library/react';
import Footer from '../index.tsx';

describe('Footer', () => {
  it('should render correctly', () => {
    const screen = render(<Footer />);
    expect(screen.asFragment()).toMatchSnapshot();
  });
});