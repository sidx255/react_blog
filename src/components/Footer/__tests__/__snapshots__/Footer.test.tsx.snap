// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<DocumentFragment>
  <div
    class="body-padding footer"
  >
    Â© Siddharth Sharma 2023
  </div>
</DocumentFragment>
`;
